xml/templates/message_templates/contribution_offline_receipt_html.tpl

```diff
@@ -131,13 +131,13 @@
       {/foreach}
      {/if}
 
-     {if isset($totalTaxAmount) && $totalTaxAmount !== 'null'}
+     {if $isShowTax}
```

Comment:
It is a little weird that it now always shows $0 if there's no tax, but I don't think it's a big deal.

```
Financial Type: Donation

Total Tax Amount : $0.00
Total Amount : $123.45
Date Received: January 21st, 2022  4:40 PM
Receipt Date: January 21st, 2022  4:41 PM
Paid By: Check
```

Reply:
Oh wait it did before too if you have the tax setting turned on. But now it's adding a time component to the dates whereas it didn't before. I don't personally have a strong opinion on that.